lib/whitehall_importer/integrity_checker.rb

```diff
@@ -2,6 +2,15 @@ module WhitehallImporter
   class IntegrityChecker
     attr_reader :edition
 
+    def self.time_similar?(proposed_time, publishing_api_time)
+      return true if proposed_time == publishing_api_time
+
+      Time.zone.rfc3339(proposed_time).between?(Time.zone.rfc3339(publishing_api_time) - 5,
+                                                Time.zone.rfc3339(publishing_api_time) + 5)
```

Comment:
There's a lot to parse in a couple of lines here perhaps it should be broken up a bit

eg:

```suggestion
      proposed_time = Time.zone.rfc3339(proposed_time)
      publishing_api_time = Time.zone.rfc3339(publishing_api_time)
      proposed_time.between?(publishing_api_time - 5, publishing_api_time + 5)
```

Reply:
Sure